### PR TITLE
Update READMEs and add SVG images.

### PR DESCRIPTION
**Pull request by ekechinwokah**
## What does this PR do?
- Update repo READMEs

## How does this PR work? (optional)
- N/A

## Refer to a related PR or issue link
- N/A

## Checklist
- N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,123 +1,103 @@
...
 
-* [Integration with Fluentbit](examples/fluentbit/README.md) - Learn how to publish logs and time series to Infino using FluentBit, a popular observability tool.
-* [LLM Monitoring with langchain](examples/llm-monitoring-langchain/llm-monitoring-langchain.ipynb) - Discover how Infino's callback in Langchain can be used for monitoring the requests in real-time. Feel free to checkout [langchain documentation](https://python.langchain.com/docs/ecosystem/integrations/infino).
-* [LLM Monitoring with OpenAI](examples/llm-monitoring-openai/llm-monitoring-openai.ipynb) - See an example of monitoring OpenAI's services using Infino's powerful observability features.
-* Using Infino's Python client - Get started with the Python client for Infino and explore its functionalities. Checkout Infino's [python package](https://pyup.io/packages/pypi/infinopy/).
+## Contact Us
 
-Please navigate to the respective example folders to find instructions and code samples for each scenario.
+Ping us on [Slack](https://infinohq.slack.com/join/shared_invite/zt-1tqqc0vsz-jF80cpkGy7aFsALQKggy8g#/shared-invite/email) or send an email to info@infino.ai.
```
Comment (vinaykakade) [inline README.md:92]: Make this info AT infino DOT ai, or something similar so that crawler bots don't get it. You can also make it to be an image instead.

File: docs/Infino_logo_dark.svg
Comment (vinaykakade) [inline docs/Infino_logo_dark.svg:1]: Should we have the text infino centrally aligned in the logo?

File: docs/Infino_logo_light.svg
Comment (vinaykakade) [inline docs/Infino_logo_light.svg:1]: Same as above

File: README.md [hunk 2]
```diff
@@ -1,123 +1,103 @@
...
+- Install [Rust toolchain](https://www.rust-lang.org/tools/install).
 
-## :punch: Contributions
 
-Contributions are welcome in a bunch of areas and highly appreciated! To get started, check out the [contributing guidelines](CONTRIBUTING.md).
+### Examples
 
-You can also join us on [Slack](https://infinohq.slack.com/join/shared_invite/zt-1tqqc0vsz-jF80cpkGy7aFsALQKggy8g#/shared-invite/email).
+* [Integration with Fluentbit](examples/fluentbit/README.md) - Learn how to publish logs and metrics to Infino using FluentBit.
+* [LLM monitoring with Langchain](examples/llm-monitoring-langchain/llm-monitoring-langchain.ipynb) - Discover how Infino's callback in Langchain can be used for monitoring requests in real-time. Also checkout the [Langchain docs](https://python.langchain.com/docs/ecosystem/integrations/infino).
```
Comment (vinaykakade) [inline README.md:77]: nit: not part of this PR's change, but while you are at it, would be good to change "LangChain docs" to "LangChain <> Infino docs" - to convey that the link is about the integration between the two.

File: README.md (hunk 1 above)
Comment (ekechinwokah) [reply README.md]: done. used image.

File: docs/Infino_logo_dark.svg
Comment (ekechinwokah) [reply docs/Infino_logo_dark.svg:1]: done. pls confirm.

File: docs/Infino_logo_light.svg
Comment (ekechinwokah) [reply docs/Infino_logo_light.svg:1]: done. pls confirm.

File: README.md (hunk 2 above)
Comment (ekechinwokah) [reply README.md]: sure thing. done.

File: .DS_Store
Comment (vinaykakade) [inline .DS_Store:1]: You can remove the .DS_Store from the commit

File: docs/Infino_logo_dark.svg
Comment (vinaykakade) [reply docs/Infino_logo_dark.svg:1]: I think the new image isn't yet pushed? Still seeing left aligned "infino" in the logo.

File: docs/Infino_logo_light.svg
Comment (vinaykakade) [reply docs/Infino_logo_light.svg:1]: I think the new image isn't yet pushed? Still seeing left aligned "infino" in the logo.

Comment (ekechinwokah) [reply docs/Infino_logo_light.svg:1]: do you mean the "infino" text within the logo is left aligned or the logo itself is left aligned?

File: docs/Infino_logo_dark.svg
Comment (ekechinwokah) [reply docs/Infino_logo_dark.svg:1]: same as below

File: .DS_Store
Comment (ekechinwokah) [reply .DS_Store:1]: done

File: docs/Infino_logo_light.svg
Comment (vinaykakade) [reply docs/Infino_logo_light.svg:1]: I meant the infino text within the logo

Comment (ekechinwokah) [reply docs/Infino_logo_light.svg:1]: i can't see what you are seeing. https://github.com/infinohq/infino/pull/125/files
